### DISCUSSION TITLE
请将仓库转移至私有仓库. Github 不是一个政治平台

### DISCUSSION THREAD

**Discussion by l-7-l** [The-Run-Philosophy-Organization/run#153] [General]
请不要利用公共平台阐述自己的政治观点。

Github 本身是无辜的 求求你 放过她吧， 自己搭一个私有仓库花不了几个钱儿， 如果因此被屏蔽将是技术人的灾难.

**Comment by the-run-philosophy** [comment]
如果技术连你的思想都无法解放，那你这技术也是白搭。
我赞同全世界向法西斯技术禁运。如你承认你是法西斯的话

**Reply by ytgui** [reply]
借楼，技术人的灾难永远不是自由的讨论，而是法不可测，铁拳悬在头顶不知何时落下

**Reply by ghost** [reply]
借楼。>请不要利用公共平台阐述自己的政治观点。
☁人真的不妙

**Reply by leon-fudan** [reply]
@bosskwei  如果这所谓铁拳让你生活水平提升，全市GDP提升，个人收入提升，基础建设排名世界靠前，生活非常便利。而且这个铁拳至少40年都没有落下，那么它会落下吗？我不懂你的逻辑。

**Comment by the-run-philosophy** [comment]
希望你自己好好反省自己匍匐苟且的习惯。

**Reply by ytgui** [reply]
请朗读并背诵：希望你自己好好反省自己匍匐苟且的习惯。

**Comment by ghost** [comment]
根据[blocky](https://blocky.greatfire.org/detail/13632/https%3A%2F%2Fgithub.com)的测试数据，Github在去年2月就遭到了SNI干扰，这个项目是这个月才搞起来的吧（笑

**Comment by NeroBlackstone** [comment]
😅为什么run也是讨论政治，我不是很懂
😅“有的人跪久了就站不起来了”

**Comment by yuchen001** [comment]
太监真多

**Comment by l-7-l** [comment]
反贼之所以上不了台面是因为不懂得**力量来源**在哪里。
其次, 这个仓库是纯技术指南类型的吗? 又是鼓吹西方政体又是要搞没有钳制的的资本市场 这不叫涉政吗

再则， 我没有对任何一个人做出人身攻击， 也不用劝我反思，我倒是建议你们好好反省 为什么搞了几十年 愣是没有成地区、成规模的群众跟着你们走？ 

最后 你们可以随意表达向所谓的"法西斯搞技术禁运" 这种类似的观点，但是 请问 **你们在"畅快呼吸"的同时为什么要伤害别人呢** 普通单纯热爱技术的人有什么错 ?

**Reply by h7221836** [reply]
技术本身是没有问题的，唉可惜人是有政治立场的

**Reply by SunsetMkt** [reply]
> 反贼之所以上不了台面是因为不懂得**力量来源**在哪里。 其次, 这个仓库是纯技术指南类型的吗? 又是鼓吹西方政体又是要搞没有钳制的的资本市场 这不叫涉政吗
> 
> 再则， 我没有对任何一个人做出人身攻击， 也不用劝我反思，我倒是建议你们好好反省 为什么搞了几十年 愣是没有成地区、成规模的群众跟着你们走？
> 
> 最后 你们可以随意表达向所谓的"法西斯搞技术禁运" 这种类似的观点，但是 请问 **你们在"畅快呼吸"的同时为什么要伤害别人呢** 普通单纯热爱技术的人有什么错 ?

**对于GitHub的干扰一直存在**，这个Repo不可能影响（至少是正向影响）GitHub的访问状态。

无论是否是纯技术指南类型，这**不违反GitHub的使用规则**（只有严重暴力、严重色情、对其它用户的骚扰是明确在社区守则里禁止的），根据个人好恶/是否是真理来决定内容是否应当放在GitHub上是不成立的（同时也无法举报成功，~~别问我为什么知道，因为我真的举报过其他的涉政库~~）。

当然，“向法西斯技术禁运”的观点本身也是“法西斯”，因为世界上的人类应当对于开源技术有同等的访问权力（一般是开源许可规定的，但可能会因为地区法律而受到阻碍，这并不是开源技术的错），技术工作者是极其容易被裹挟着的，更容易成为受害者，这种观点当然**应当批判**。

对于“为什么要伤害别人”的问题，只能不恰当地说，不是所有人都有足够高的道德水平，但您不可能要求所有人都像您一样有极高的“敏感性”或者是“自控性”（我们暂且不论这“自控”是否是应该/合理的），而且正如第一段所述，GitHub早已被干扰，这并不是因为这个Repo。只要不违反社区守则，原则上这里可以放任何东西，而在这样的境外网站中，GitHub是个（没被彻底阻止的）**特例**，这不是普遍现象。

**我非常同情、也同样遭受着您所遭受的困扰**，但这不可能通过您一个人/几个人的努力和呼吁解决，也根本不可能是个简单的问题。万分抱歉。

**Comment by yuchen001** [comment]
技术本来就自带立场了，这点不用遮掩，人就生活在政治当中。

Lubb ***@***.***> 于2022年4月17日周日 16:56写道：

**Comment by williamtuw** [comment]
这样下去github 肯定没了 ，  网信办你饶了github 吧，要严厉惩治这些反贼才行

**Reply by ghost** [reply]
> 这样下去github 肯定没了 ， 网信办你饶了github 吧，要严厉惩治这些反贼才行

引用他人观点：“就算网信办因为这个repo把github封了，你不去骂网信办，骂这个repo？”

**Reply by l-7-l** [reply]
反贼都算不上，反贼是要掀翻暴力机器的，反的是政府 这群人和轮子没什么两样 反的是人人民 讨厌这片土地的自己走就是了 不是自诩清高吗 自己搞一个服务器 干干净净的不给人添麻烦多好 有些牧羊犬不是天天吹倭奴和洋大人素质高 不给人添麻烦吗 绝了也是 素质又低 言行又不一致 这群垃圾最好全滚出去

> 这样下去github 肯定没了 ，  网信办你饶了github 吧，要严厉惩治这些反贼才行

**Reply by ghost** [reply]
> 反贼都算不上，反贼是要掀翻暴力机器的，反的是政府 这群人和轮子没什么两样 反的是人人民 讨厌这片土地的自己走就是了 不是自诩清高吗 自己搞一个服务器 干干净净的不给人添麻烦多好 有些牧羊犬不是天天吹倭奴和洋大人素质高 不给人添麻烦吗 绝了也是 素质又低 言行又不一致 这群垃圾最好全滚出去
> 
> > 这样下去github 肯定没了 ，  网信办你饶了github 吧，要严厉惩治这些反贼才行

您看过轮子的宣传吗？这个项目这么就反起政府来了？
而且Github早在去年2月份就遭到GFW的干扰，而且Github上比这个反动的项目和用户多了去了，为什么您就来这里开骂？
难道在中国就必须得有言论审查吗？人大代表中有多少资产阶级和不信马列的人？在我国劳动法是有用还是个摆设？
这次上海的种种魔幻行为您有所了解吗？[上海的警察真动手打人了您有所了解吗？](https://twitter.com/lidangzzz/status/1514513628090834946)

这到底是谁在反对人民？这到底谁站在了人民的对立面？

**Reply by l-7-l** [reply]
> > 反贼都算不上，反贼是要掀翻暴力机器的，反的是政府 这群人和轮子没什么两样 反的是人人民 讨厌这片土地的自己走就是了 不是自诩清高吗 自己搞一个服务器 干干净净的不给人添麻烦多好 有些牧羊犬不是天天吹倭奴和洋大人素质高 不给人添麻烦吗 绝了也是 素质又低 言行又不一致 这群垃圾最好全滚出去
> 
> > 
> 
> > > 这样下去github 肯定没了 ，  网信办你饶了github 吧，要严厉惩治这些反贼才行
> 
> 
> 
> 您看过轮子的宣传吗？这个项目这么就反起政府来了？
> 
> 而且Github早在去年2月份就遭到GFW的干扰，而且Github上比这个反动的项目和用户多了去了，为什么您就来这里开骂？
> 
> 难道在中国就必须得有言论审查吗？人大代表中有多少资产阶级和不信马列的人？在我国劳动法是有用还是个摆设？
> 
> 这次上海的种种魔幻行为您有所了解吗？[上海的警察真动手打人了您有所了解吗？](https://twitter.com/lidangzzz/status/1514513628090834946)
> 
> 
> 
> 这到底是谁在反对人民？这到底谁站在了人民的对立面？

兴百姓苦，亡百姓更苦。我认可这片土地上的所有政权与人民，一个能认错的政府在我眼里就是好政府，天下没有完美的体制，任何国家都有优缺点 重要的是能不能改，现在上头风向已经转了，一切都是时间问题。

别人骂的你看到了没？ 一句正经话没有 长了个脑子就知道骂人 三代贫农给干成了跪地太监法西斯 这是不是素质低下？ 骂得有错吗？ 不能这么双标啊

**Reply by ghost** [reply]
> > > 反贼都算不上，反贼是要掀翻暴力机器的，反的是政府 这群人和轮子没什么两样 反的是人人民 讨厌这片土地的自己走就是了 不是自诩清高吗 自己搞一个服务器 干干净净的不给人添麻烦多好 有些牧羊犬不是天天吹倭奴和洋大人素质高 不给人添麻烦吗 绝了也是 素质又低 言行又不一致 这群垃圾最好全滚出去
> > 
> > 
> > > 
> > 
> > 
> > > > 这样下去github 肯定没了 ，  网信办你饶了github 吧，要严厉惩治这些反贼才行
> > 
> > 
> > 您看过轮子的宣传吗？这个项目这么就反起政府来了？
> > 而且Github早在去年2月份就遭到GFW的干扰，而且Github上比这个反动的项目和用户多了去了，为什么您就来这里开骂？
> > 难道在中国就必须得有言论审查吗？人大代表中有多少资产阶级和不信马列的人？在我国劳动法是有用还是个摆设？
> > 这次上海的种种魔幻行为您有所了解吗？[上海的警察真动手打人了您有所了解吗？](https://twitter.com/lidangzzz/status/1514513628090834946)
> > 这到底是谁在反对人民？这到底谁站在了人民的对立面？
> 
> 兴百姓苦，亡百姓更苦。我认可这片土地上的所有政权与人民，一个能认错的政府在我眼里就是好政府，天下没有完美的体制，任何国家都有优缺点 重要的是能不能改，现在上头风向已经转了，一切都是时间问题。
> 
> 别人骂的你看到了没？ 一句正经话没有 长了个脑子就知道骂人 三代贫农给干成了跪地太监法西斯 这是不是素质低下？ 骂得有错吗？ 不能这么双标啊

的确如此，但人应该有选择跑路的权限（
而且网信办是不会封掉Github的，有这种想法的这是政治在搞你，而不是在搞Github

**Reply by ghost** [reply]
> > > 反贼都算不上，反贼是要掀翻暴力机器的，反的是政府 这群人和轮子没什么两样 反的是人人民 讨厌这片土地的自己走就是了 不是自诩清高吗 自己搞一个服务器 干干净净的不给人添麻烦多好 有些牧羊犬不是天天吹倭奴和洋大人素质高 不给人添麻烦吗 绝了也是 素质又低 言行又不一致 这群垃圾最好全滚出去
> > 
> > 
> > > 
> > 
> > 
> > > > 这样下去github 肯定没了 ，  网信办你饶了github 吧，要严厉惩治这些反贼才行
> > 
> > 
> > 您看过轮子的宣传吗？这个项目这么就反起政府来了？
> > 而且Github早在去年2月份就遭到GFW的干扰，而且Github上比这个反动的项目和用户多了去了，为什么您就来这里开骂？
> > 难道在中国就必须得有言论审查吗？人大代表中有多少资产阶级和不信马列的人？在我国劳动法是有用还是个摆设？
> > 这次上海的种种魔幻行为您有所了解吗？[上海的警察真动手打人了您有所了解吗？](https://twitter.com/lidangzzz/status/1514513628090834946)
> > 这到底是谁在反对人民？这到底谁站在了人民的对立面？
> 
> 兴百姓苦，亡百姓更苦。我认可这片土地上的所有政权与人民，一个能认错的政府在我眼里就是好政府，天下没有完美的体制，任何国家都有优缺点 重要的是能不能改，现在上头风向已经转了，一切都是时间问题。
> 
> 别人骂的你看到了没？ 一句正经话没有 长了个脑子就知道骂人 三代贫农给干成了跪地太监法西斯 这是不是素质低下？ 骂得有错吗？ 不能这么双标啊

哦对了关于政府认错：所以我国政府什么时候会为软禁赵紫阳而认错呢？什么时候能不搞“博主已开启精选评论”？什么时候会查清彭帅事件的真相？什么时候能真正让人民群众说话？

**Reply by yuchen001** [reply]
> 反贼都算不上，反贼是要掀翻暴力机器的，反的是政府 这群人和轮子没什么两样 反的是人人民 讨厌这片土地的自己走就是了 不是自诩清高吗 自己搞一个服务器 干干净净的不给人添麻烦多好 有些牧羊犬不是天天吹倭奴和洋大人素质高 不给人添麻烦吗 绝了也是 素质又低 言行又不一致 这群垃圾最好全滚出去
> 
> > 这样下去github 肯定没了 ，  网信办你饶了github 吧，要严厉惩治这些反贼才行

粉红读过书没？写字请把标点符号加上。没文化的破玩意，还来别人的仓库里指点江山，政治老师当上瘾了？

**Reply by l-7-l** [reply]
> > 反贼都算不上，反贼是要掀翻暴力机器的，反的是政府 这群人和轮子没什么两样 反的是人人民 讨厌这片土地的自己走就是了 不是自诩清高吗 自己搞一个服务器 干干净净的不给人添麻烦多好 有些牧羊犬不是天天吹倭奴和洋大人素质高 不给人添麻烦吗 绝了也是 素质又低 言行又不一致 这群垃圾最好全滚出去
> 
> > 
> 
> > > 这样下去github 肯定没了 ，  网信办你饶了github 吧，要严厉惩治这些反贼才行
> 
> 
> 
> 粉红读过书没？写字请把标点符号加上。没文化的破玩意，还来别人的仓库里指点江山，政治老师当上瘾了？

你骂谁？ 有种就说出来

**Comment by HendricksJudy** [comment]
那肯定是大力支持封禁GitHub了，地球上少了太监还能转不起来🐴。看着就乐，自己没有只能用人家的还被自己人掐断的场面可真滑稽。

**Reply by leon-fudan** [reply]
人家的？你在创建的时候可不会标榜是美国的，二标榜是世界的。
这么多中国人在github 上开源贡献了自己的代码，说是你的就是你的了？

**Comment by sakurashino** [comment]
不要和粉红五毛有任何争论，只会浪费口舌

**Reply by leon-fudan** [reply]
没有人是粉红，没有人是五毛，只是看不得一群人在诅咒自己生活的土地，甚至引导人们怀疑自己的种族。

**Reply by ytgui** [reply]
> 没有人是粉红，没有人是五毛，只是看不得一群人在诅咒自己生活的土地，甚至引导人们怀疑自己的种族。

小伙子说话这么豪横，想必您就是羊上羊或者资本家吧？

**Reply by leon-fudan** [reply]
随便扣帽子真可笑。不过听你说资本家我是真的笑了。
你知道你要run的目的地社会叫资本主义社会吗？而且资本主义社会不是别人给扣的帽子，是共识。

**Comment by yuliu-wish** [comment]
你妈贵姓？
